src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/MSITest.java

```diff
@@ -299,7 +300,9 @@ public void testNumericAkvWithBadCred() throws SQLException {
             testNumericAKV(connStr);
             fail(TestResource.getResource("R_expectedFailPassed"));
         } catch (Exception e) {
-            assert (e.getMessage().contains("AuthenticationException"));
+            assertTrue(e.getCause() instanceof MsalServiceException);

```

Comment:
What is the actual exception type here? is it being thrown as SQLServerException?